requirements-base.txt

```diff
@@ -52,7 +52,7 @@ rfc3339-validator==0.1.2
 rfc3986-validator==0.1.1
 # [end] jsonschema format validators
 sentry-relay>=0.8.1,<0.9.0
-sentry-sdk>=0.17.7,<0.18.0
+sentry-sdk @ git+https://github.com/getsentry/sentry-python@c277ed5#egg=sentry-sdk
```

Comment:
This prevents us from deploying to PyPI so I'll have to block this PR unfortunately.